Perlang.ConsoleApp/Perlang.ConsoleApp.csproj

```diff
@@ -24,9 +23,14 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <!-- Assemblies implicitly referenced via reflection must be listed here, to be included in the
-             trimmed result -->
+        <!-- Assemblies implicitly referenced via reflection must be listed here, to be included in the trimmed result.
+             If you get an error about 'Could not load type 'Foo' from assembly 'Bar', add the assembly for the _Foo_
+             type to this list. -->
         <TrimmerRootAssembly Include="System.Runtime.Extensions" />
+        <TrimmerRootAssembly Include="System.Runtime.InteropServices" />
+
+        <!-- Certain assemblies generate errors on R2R compilation. We run them in JIT mode for now. -->
+        <PublishReadyToRunExclude Include="System.Diagnostics.Process.dll" />
```

Comment:
This was an important part of getting this fixed. This fixes the error described in the PR description for #126. With this in place, the code would compile.

...however, we still got a runtime error because of an attribute having being trimmed away from `System.Runtime.InteropServices`. That's why we also needed to exclude that assembly from trimming altogether.